CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,37 @@
+In order to keep the DevelopersIndia community warm, welcoming, and safe place we ask that all participants read and adhere to the following code of conduct.
+
+# Rules
+* Be kind and courteous to your fellow members.
+* Criticize ideas, not people.
+* Discuss publicly. Refrain from direct messaging people unless it is specifically requested.
```

Comment:
Discuss publicly & refrain.... (omit the period & insert "_and_")

Reply:
done